content/workshops/openshift_101_dcmetromap/lab3-s2i.md

```diff
@@ -18,8 +18,8 @@ We can do this either via the command line or the web console.  You decide which
 <i class="fa fa-terminal"></i> Goto the terminal and type the following:
 </blockquote>
 
-```
-$ oc new-app --name=dc-metro-map https://github.com/dudash/openshift-workshops.git --context-dir=dc-metro-map
+```bash
+$ oc new-app --name=dc-metro-map https://github.com/RedHatGov/openshift-workshops.git --context-dir=dc-metro-map
```

Comment:
I'm OK with this change but we need to sync this fork with my last 3 commits from dudash/master to RedHatGov/master - 1 of those changes is critical.

Also, longer term I suggest we create a new repo in RedHatGov for this app.  OR we could setup a shared repo for all workshop code.  I think that's probably good for discussion during the next weekly meeting, there are pros/cons for each.